app/build.gradle

```diff
@@ -165,6 +168,12 @@ android {
             versionName computeVersionName('greatfire')
             // signingConfig signingConfigs.prod
         }
+        fdroid {
+            applicationId 'org.greatfire.wikiunblocked'
```

Comment:
Since the f-droid.org version will be signed by the f-droid.org APK signing key, I recommend using a specific Application ID so there isn't update/signature conflicts:
```suggestion
            applicationId 'org.greatfire.wikiunblocked.fdroid'
```

Reply:
it seems like the best way to fix this is to remove the ".fdroid" suffix when i fetch the secrets in the code.  i couldn't find a way to reference a different set of native files for each build flavor.